### ISSUE TITLE
Colored texts on inverted/darkmode not complient

### ISSUE THREAD

**Issue by eduardodir** [fremtind/jokul#896]
**Describe the bug**
Any type of text in our "colors" success, error etc... will not pass web accessibility as seen in this example if they are used under input or standalone on black or very dark background color.

**To Reproduce**
1. Go to Select component in the portal
2. select error and inverted from the options. See picture.

**Expected behavior**
A color that was clear to read and passed web accessibility or a background color that made it pass.

**Screenshots**

**Desktop (please complete the following information):**

-   Macbook, Chrome

**Additional context**
<img width="627" alt="Screenshot 2020-04-13 at 22 21 17" src="https://user-images.githubusercontent.com/56429532/79158078-faf3a900-7dd5-11ea-9553-9c44b3b963e6.png">

**Comment by Mikkelinski** [comment]
Error tekst for "inverted" (dark) skal ikke være #AA1F23 men i #FF8B79

**Comment by Mikkelinski** [comment]
@piofinn er denne løst enda? Altså utviklet med riktig farge

**Comment by piofinn** [comment]
Det er fikset i `Label`-komponenten, men må aktivt skrus på i de komponentene som bruker den.

**Comment by eduardodir** [comment]
@piofinn Kan vi skapa en ny issue för å "aktivt skrus på i de komponentene som bruker den"?

**Comment by piofinn** [comment]
@eduardodir Vi bør vel egentlig ha et issue for "implementer dark mode i alle komponenter", hvor riktig bruk av `SupportLabel` blir en del av det. Jeg kan lage en milestone for det!

**Comment by Mikkelinski** [comment]
@piofinn er dette fortsatt et aktuelt issue eller kan jeg lukke?

**Comment by Mikkelinski** [comment]
<img width="708" alt="Screenshot 2020-06-05 at 10 18 59" src="https://user-images.githubusercontent.com/8458054/83853733-ff717980-a715-11ea-9bef-1904b420e38d.png">
Ser ut som denne fortsatt er feil?
